content/20.results.md

```diff
@@ -89,9 +89,15 @@ Interaction terms did not predict the group of scientists over and above the mai
 
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/region_breakdown.png){#fig:region_breakdown}
 
-When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.18, $\beta$~East\ Asian~ = -1.71, _p_ = 6.5×10^-4^).
-We observed no statistically significant difference between the proportion of honors given to authors with Celtic/English names, European names, or names in Other categories (_p_ = 0.61, _p_ = 0.12, and _p_ = 0.72, respectively).
-Please see [analysis notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html) for more details.
+When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.17, $\beta$~East\ Asian~ = -1.77, _p_ = 4.3×10^-4^).
+We observed no statistically significant difference between the proportion of honors given to authors with Celtic/English names, European names, or names in Other categories (_p_ = 0.60, _p_ = 0.11, and _p_ = 0.70, respectively).
+Please see the [US analysis notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html) for more details.
+
+An alternative approach that considered each pair of forename and last name as the unit of measurement showed similar results.
```

Comment:
We will need to write an extremely short methods paragraph about this. Also, we should probably expand the considerations paragraph to discuss why we don't use this strategy as the primary approach:
* selecting an honoree a second time really is a distinct selection process
* name collisions are hazardous to analyses and tools like ORCID are not widespread enough to be used for this purpose yet.